ApiTransform.ps1

```
@@ -611,7 +617,7 @@ Function Update-SwaggerObject {
                         } Else {
                             $excludePath = ($InputObjectName -replace '.paths.', '') -replace "\.\w+", ''
                             # Write-Warning "[status] New SDK Endpoint Found: `nOperationId: $($ThisObject.operationId) `nPath: $excludePath"
-                            Write-Error ("In '$($CurrentSDKName)' unknown operationId '$($ThisObject.operationId) - $($InputObjectName)'.")
+                            Write-Warning ("In '$($CurrentSDKName)' unknown operationId '$($ThisObject.operationId) - $($InputObjectName)'.")

```

This really didn't need to throw an error/ switched to write-warning